randnum.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+int main()
+{
+	srand(time(0));
+	int r = rand() % (10 + 1 - 1) + 1;
```

Comment:
This is mostly correct! I'm not sure why you bothered with the + 1 -1 portion - that can just reduce down. With that said, you still did it correctly and efficiently, so good job! Also want to note that you should add comments for good programming practice.